sycl/include/sycl/detail/group_sort_impl.hpp

```diff
@@ -365,9 +365,9 @@ convertToOrdered(ValT value) {
 // converts floating type to Ordered (in terms of bitness) type
 template <typename ValT>
 std::enable_if_t<!std::is_same<ValT, OrderedT<ValT>>::value &&
-                         std::is_floating_point<ValT>::value ||
-                     std::is_same<ValT, sycl::half>::value ||
-                     std::is_same<ValT, sycl::ext::oneapi::bfloat16>::value,
+                     (std::is_floating_point<ValT>::value ||
+                      std::is_same<ValT, sycl::half>::value ||
+                      std::is_same<ValT, sycl::ext::oneapi::bfloat16>::value),
```

Comment:
Don't want to block this, but we have `std::is_same_v` from C++17 unconditionally now.

Reply:
Done.